src/lib/loadBatchPriceInsights.ts

```diff
@@ -0,0 +1,47 @@
+import gql from "lib/gql"
+import { extractNodes } from "lib/helpers"
+
+const MAX_MARKET_PRICE_INSIGHTS = 50
+
+export const loadBatchPriceInsights = async (
+  artistIDMediumTuples: any[],
+  vortexGraphQLLoader: any
+) => {
+  try {
+    const vortexResult = await vortexGraphQLLoader({
+      query: gql`
+        query marketPriceInsightsBatchQuery($artistIDMediumTuples: [ArtistIdMediumTupleType!]!) {
```

Comment:
https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#query-naming-conventions

```suggestion
        query MarketPriceInsightsBatchQuery($artistIDMediumTuples: [ArtistIdMediumTupleType!]!) {
```